CMakeLists.txt

```diff
@@ -58,6 +58,24 @@ __sync_add_and_fetch(&a, 1);
 return 0;
 }" ARCH_SUPPORTS_64BIT_ATOMICS)
 
+FIND_LIBRARY(LIBC_BACKTRACE_LIB "execinfo")
+IF (LIBC_BACKTRACE_LIB)
+    SET(CMAKE_REQUIRED_LIBRARIES ${LIBC_BACKTRACE_LIB})
+    LIST(APPEND EXTRA_LIBS ${LIBC_BACKTRACE_LIB})
```

Comment:
Would it possible to move this line to within the `IF (LIBC_HAS_BACKTRACE)` block so that it would only be executed when `backtrace_symbols_fd` is actually being used?